Transmission.xcodeproj/project.pbxproj

```diff
@@ -3033,6 +2989,25 @@
 			shellPath = /bin/sh;
 			shellScript = "sed 's|@LIBPSL_[A-Z_]*@|0|' < third-party/libpsl/include/libpsl.h.in > third-party/libpsl/include/libpsl.h\n\n# Generate files to be included\nPYTHON=$( command -v python3 ) || PYTHON=$( command -v python3.7 ) || PYTHON=$( command -v python2 )\n\"${PYTHON}\" \"third-party/libpsl/src/psl-make-dafsa\" --output-format=cxx+ \"third-party/libpsl/list/public_suffix_list.dat\" \"third-party/suffixes_dafsa.h\"\n";
 		};
+		C82B3002295070C40001BD6E /* Swiftlint Script */ = {
+			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Swiftlint Script";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "export PATH=\"$PATH:/opt/homebrew/bin\"\nif which swiftlint > /dev/null; then\n  swiftlint --config macosx/.swiftlint.yml\nelse\n  echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
```

Comment:
I see there is a separate PR open for SwiftLint: https://github.com/transmission/transmission/pull/4757
This PR got really big so it would be nice to remove swiftlint logic from here, I will still leave some thoughts here: 
1. Consider moving this script to a separate file for readability.
1. It might be a good idea to pin the swiftlint version, right now this build step grabs any local version which might lead to inconsistent lint warnings across developer machines
1. Followup for the previous point – it would be nice to check swiftlint on CI too, right now this script just silently passes if swiftlint is not installed locally, without a CI check warnings will accumulate in the main branch over time.

Reply:
swiftlint is not enforced for now because I want the build to be as easy as possible (which is tricky with `ninja`, to a point you can't imagine: it's like if ninja only supports "objc calling swift" but not "swift calling objc"). Once all build issues are solved, we can consider make it more strict.